packages/editor/src/components/post-permalink/index.js

```diff
@@ -147,6 +148,7 @@ export default compose( [
 			permalinkParts: getPermalinkParts(),
 			postSlug: getEditedPostAttribute( 'slug' ),
 			isEditable: isPermalinkEditable(),
+			isFrontPage: isFrontPage(),
```

Comment:
It seems like this could follow other props and use `getEditedPostAttribute( 'is_front_page' )` directly.